### ISSUE TITLE
Migration of data from existing SIS-es to MeetPlan

### ISSUE THREAD

**Issue by mytja** [MeetPlan/MeetPlanFrontend#39]
Source: https://www.reddit.com/r/Slovenia/comments/vhnd34/comment/id9upuh/

I do not have access to any of these exports or imports, so if you have it, it would be greatly appreciated.

**Comment by aikenahac** [comment]
I'm 99.9999999% sure it's not legal for someone to give you exported data from their school, as it contains a lot of personal information about their students and staff.

**Comment by mytja** [comment]
> I'm 99.9999999% sure it's not legal for someone to give you exported data from their school, as it contains a lot of personal information about their students and staff.

Hello, and thanks for your comment. I'm not asking for full exports that contain personal and/or sensitive data - far from that - all I want are the rough CSV (or whatever format) keys and possibly some (max. 5) entries with replaced sensitive data with example data (for example, replace students' names with "Janez Primer" or something similar)

**Comment by aikenahac** [comment]
Yes, that makes more sense, but there might be a contractual obligation from the schools preventing them from sharing even the structure, such as an NDA (Non-Disclosure Agreement).

**Comment by mytja** [comment]
> there might be a contractual obligation from the schools preventing them from sharing even the structure, such as an NDA (Non-Disclosure Agreement).

I am presently not aware of anything like that, but I don't have any proper paperwork/NDAs/contracts by eAsistent or Lo.Polis (as I'm not a school), but you probably know better (since you seem to be working for eAsistent).

**Comment by aikenahac** [comment]
My point is, this would be tricky to implement.
While I work for eAsistent, I do not have access to any backend code as I am a student. I also don't have access to any paperwork with the schools since I'm in the development department, so everything I say here is based on my assumptions from studying a bit of basic business law.
